src/fetch.ts

```diff
@@ -44,6 +44,18 @@ async function downloadChromeFile(url: string) {
     }
 }
 
+async function addCustomSchemas() {
+    try {
+        const files = fs.readdirSync("./schemas-custom");
```

Comment:
schemas-custom doesn't sound right to me.. what do you think about "schemas-additional"?

Reply:
Forgive my poor English... 
I have updated the pull request according to your suggestion. Feel free to review it.
hope you recover soon.